### PR TITLE
Add single_line_after_imports fixer

### PR DESCRIPTION
**Pull request by ceeram**
Replaces #816
Implements #153

### CODE REVIEW

File: Symfony/CS/Fixer/PSR2/SingleLineAfterImportsFixer.php [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+        return $tokens->generateCode();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDescription()
+    {
+        return 'Each namespace use MUST go on its own line and there MUST be one blank line after the use statements block.';
```
Comment (stof) [inline Symfony/CS/Fixer/PSR2/SingleLineAfterImportsFixer.php]: technically, only the second part of the description is actually describing this fixer. The first part is describing another fixer.

Comment (keradus) [inline Symfony/CS/Fixer/PSR2/SingleLineAfterImportsFixer.php]: Nice catch. Could you provide a PR with better description ?

Comment (ceeram) [inline Symfony/CS/Fixer/PSR2/SingleLineAfterImportsFixer.php]: @stof i think you are mixing up with:

There MUST be one use keyword per declaration.
use One, Two

Indeed there is a fixer for that already


This fixer is about:
use One; use Two;

Comment (stof) [inline Symfony/CS/Fixer/PSR2/SingleLineAfterImportsFixer.php]: ``use One; use Two;`` would be about the rule ``There MUST be only one statement per line``

Comment (ceeram) [inline Symfony/CS/Fixer/PSR2/SingleLineAfterImportsFixer.php]: How is ` There MUST be only one statement per line` different from `Each
namespace use MUST go on its own line` ?

2015-01-03 3:15 GMT+01:00 Christophe Coevoet <notifications@github.com>:

Comment (ceeram) [inline Symfony/CS/Fixer/PSR2/SingleLineAfterImportsFixer.php]: This fixer combines section 2.3 https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#23-lines
`There MUST NOT be more than one statement per line.`

As said before this fixer is not about section 3:
`There MUST be one use keyword per declaration.`
That is already handled by https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/Symfony/CS/Fixer/PSR2/MultipleUseFixer.php#L85

with:

section 3 https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#3-namespace-and-use-declarations
`There MUST be one blank line after the use block.`

Comment (ceeram) [inline Symfony/CS/Fixer/PSR2/SingleLineAfterImportsFixer.php]: Updated previous comment on GH

Comment (ceeram) [inline Symfony/CS/Fixer/PSR2/SingleLineAfterImportsFixer.php]: All i did is translate `There MUST be only one statement per line` to `Each namespace use MUST go on its own line`
as this fixer does not handle all cases of one statement per line, it only takes care of namespace use statements. If you think you prefer the more generic text, feel free to change it, but imo the current text is a better suit.

**Pull request closed without merging**